docs/examples/timer_operations.rst

```diff
@@ -0,0 +1,54 @@
+Timer Operations Script
+-----------------------
+
+This example demonstrates the usage of methods on the Timer client. We will assume that
+valid access tokens for the Timer and Transfer services have already been exported as
+``TIMER_TOKEN`` and ``TRANSFER_TOKEN`` environment variables, respectively.
+
+.. code-block:: python
+
+    import os
+    from datetime import datetime
+
+    from globus_sdk import (
+        AccessTokenAuthorizer,
+        TimerClient,
+        TimerJob,
+        TransferClient,
+        TransferData,
+    )
+
+    GOEP1 = "ddb59aef-6d04-11e5-ba46-22000b92c6ec"
+    GOEP2 = "ddb59af0-6d04-11e5-ba46-22000b92c6ec"
+    transfer_token = os.getenv("TRANSFER_TOKEN")
+    timer_token = os.getenv("TIMER_TOKEN")
+
+    transfer_authorizer = AccessTokenAuthorizer(transfer_token)
+    transfer_client = TransferClient(authorizer=transfer_authorizer)
+    data = TransferData(transfer_client, GOEP1, GOEP2)
+    data.add_item("/share/godata/file1.txt", "/~/file1.txt")
+
+    timer_authorizer = AccessTokenAuthorizer(timer_token)
+    timer_client = TimerClient(authorizer=timer_authorizer)
+
+    start = datetime.utcnow()
+    # We're going to run this job only a single time (``stop_after_n=1``), so the
+    # interval must be specified as ``None``, meaning it does not repeat. If we were
+    # going to set up a recurring job, we would use, for example,
+    # ``interval = timedelta(days=7)``.
+    interval = None
+    name = "example-timer-job"
+    job = TimerJob.from_transfer_data(data, start, interval, stop_after_n=1, name=name)
```

Comment:
After discussion, we agree that this might "explain better" if in the example,

```
interval = timedelta(minutes=30)
stop_after_n = 2
```

so that we don't need to explain the `n=1` case here.

But wait, there's more! We should also explain the `n=1` behavior in the TimerJob docs.